### DISCUSSION TITLE
Camera limits don't seem to work in exported game?

### DISCUSSION THREAD

**Discussion by TaylorAnderson** [ramokz/phantom-camera#235] [Question / Help Wanted]
I just started using this plugin, and am loving it so far.  However, when I export the game, the camera acts very differently than it did in the editor -- specifically, it seems to be ignoring the limits I set for it.  Do I need to do anything to get it working in the export? Gifs attached for example.
![GIF 3-20-2024 9-54-34 AM](https://github.com/ramokz/phantom-camera/assets/5308535/577dc531-56bd-421c-a992-4d9a56697bc1)

![GIF 3-20-2024 10-39-48 AM](https://github.com/ramokz/phantom-camera/assets/5308535/1604490d-79cd-4b16-8f0b-06e693000488)

**Comment by ramokz** [comment]
This is a known issue (#164), and, unfortunately, I don't know what exactly causes it. In my tests, it works correctly about 20% of the time, which suggests it's an export bug. But again, don't know why it happens.

That said, and as mentioned in the issue linked above, this seems to be solved in the `0.7` branch thanks to the refactored property system.

You can pull that branch, which _should_ make it work, but cannot guarantee it will not have any breaking changes until it's officially released.
